distributed-commandbus-jgroups/src/main/java/org/axonframework/jgroups/commandhandling/JoinMessage.java

```diff
@@ -41,6 +44,8 @@
     private Address address;
     private int loadFactor;
 
+    private static final Logger logger = LoggerFactory.getLogger(JoinMessage.class);
```

Comment:
A nit, but could you move the logger to the top of the class?

Reply:
OK, I will.